dashboard/client/src/pages/overview/cards/NodeCountCard.tsx

```diff
@@ -51,7 +53,9 @@ export const NodeCountCard = ({ className }: NodeCountCardProps) => {
   const grafanaDefaultDashboardUid =
     dashboardUids?.default ?? "rayDefaultDashboard";
   const path = `/d-solo/${grafanaDefaultDashboardUid}/default-dashboard?orgId=1&theme=light&panelId=24&var-datasource=${dashboardDatasource}`;
-  const timeRangeParams = "&from=now-30m&to=now";
+  const timeRangeParams = `&from=${job ? job.start_time : "now-30m"}&to=${
+    job ? job.end_time || "now" : "now-30m"
+  }`;
```

Comment:
sorry for responding so late, I actually meant to take in start time and end time as parameters to the NodeCountCard. This is more general than taking in a job object.